src/getCypressElementCoordinates.ts

```diff
@@ -80,26 +96,17 @@ function getIframesPositionShift(element: HTMLElement) {
   }
 
   // eslint-disable-next-line prefer-const
-  const iframes = []
-
-  while (
-    currentWindow &&
-    currentWindow !== window.top
-  ) {
-    iframes.push(
-      // for cross origin domains .frameElement returns null so query using parentWindow
-      // but when running using --disable-web-security it will return the frame element
-      (currentWindow.frameElement as HTMLElement) ??
-        currentWindow.parent.document.querySelector("iframe")
-    );
+  const iframes = [];
 
+  while (currentWindow !== window.top) {
+    iframes.push(getFrameElement(currentWindow));
     currentWindow = currentWindow.parent;
   }
 
-  return iframes.reduceRight(({ frameX, frameY, frameScale }, frame) => { 
+  return iframes.reduceRight(({ frameX, frameY, frameScale }, frame) => {
```

Comment:
Why this line changed?

Reply:
Oh, I haven't noticed this - it's my VScode being configured to strip trailing whitespace. I can revert this change if you want.

I can also setup Prettier in a separate PR if you like that tool.

Reply:
No worries, will do this myself :) 
I need to put more time into this project – I didn't expect a lot of external contributions here. We need better CI checks, prettier, git hooks etc ;)